pkg/ui/src/components/NoteButton/NoteButton.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+
+import ArrowCircleLeft from "@assets/ArrowCircleLeft.svg";
+import ArrowCircleRight from "@assets/ArrowCircleRight.svg";
+
+// #region types
+export interface NoteButtonProps
+  extends React.HTMLAttributes<HTMLButtonElement> {
+  type?: NoteButtonType;
+  label?: NoteButtonLabel | Warehouse;
+  onClick?: () => void;
+}
+
+export enum NoteButtonLabel {
+  RemoveBooks = "Remove Books",
+}
+
+export enum NoteButtonType {
+  In = "in",
+  Out = "out",
+}
+
+export enum Warehouse {
+  Scolastica = "scolastica",
+  Nuovo = "nuovo",
+  Varia = "varia",
+}
```

Comment:
Warehouses are not limited to these names, they can openly be defined by the users

Reply:
I completely agree and have mentioned it in #46, but would really love to merge this in ASAP, so that we can possibly add the updates/corrections on svelte-rewritten version of this component.